### DISCUSSION TITLE
Swipe Tab Bar Smartphone

### DISCUSSION THREAD

**Discussion by kilian-f-za** [Zefau/ioBroker.jarvis#1791] [General]
Ich habe auf einem Tab das Map Modul im Vollbild. Wenn ich nach links wischen komme ich in den Tab links. Kann man das swipen deaktivieren?

**Comment by mcuiobroker** [comment]
Version? v3.0.11 Nein, v3.1.0-alpha.27 sollte funktionieren. siehe Einstellungen.

**Reply by kilian-f-za** [reply]
Wo kann ich die Version updaten? In den Releases steht die nicht drin. Muss ich die über URL downloaden?

**Reply by mcuiobroker** [reply]
Also es noch alpha-Version mit vielen Problemen!
Update möglich über Katze und von Github und dann jarvis auswählen.

**Reply by mcuiobroker** [reply]
Einstellungen / SideBar und TabsEinstellungen
![image](https://user-images.githubusercontent.com/84818934/181796635-bbdd23ed-f2b8-4dcf-adc4-afc197427954.png)

**Reply by mcuiobroker** [reply]
Das Programm setDefaultPosition läuft da aber nicht mehr oder noch nicht! Es wird bei Änderung der Daten nicht aktualisiert.

**Reply by kilian-f-za** [reply]
> Das Programm setDefaultPosition läuft da aber nicht mehr oder noch nicht! Es wird bei Änderung der Daten nicht aktualisiert.

OK. Wird es sowas wieder geben?

**Reply by mcuiobroker** [reply]
Wird dann wieder funktionieren, wenn das Aktualisierungs-Problem gelöst ist. Da habe ich keinen Einfluss.
Wie läuft die neue Version v1.0.1?

**Reply by kilian-f-za** [reply]
Welches Aktualisierungsproblem? Von den Script?

**Reply by mcuiobroker** [reply]
Bei widget-Änderung wird nicht aktualisiert.

**Reply by kilian-f-za** [reply]
> Wird dann wieder funktionieren, wenn das Aktualisierungs-Problem gelöst ist. Da habe ich keinen Einfluss.
> Wie läuft die neue Version v1.0.1?

Ganz gut. Konnte noch nicht viel testen. Bei den Zoomstufen stimmt manchmal der Zoom nicht. Aber ich beobachte erstmal und melde mich

**Reply by kilian-f-za** [reply]
> Bei widget-Änderung wird nicht aktualisiert.

Wie? Welche Änderung?

**Reply by mcuiobroker** [reply]
Bei jeder Änderung durch das Script wird das angegebene Widget geändert. -> defaultPosition und defaultZoom. 
Dies wird aber mit v3.1.0-alpha.27 nicht im Frontend umgesetzt.

**Reply by kilian-f-za** [reply]
OK jetzt hab ich es verstanden. Also muss es da anders umgesetzt werden?

**Reply by mcuiobroker** [reply]
Nein, man kann da nichts anders umsetzen. Man muss warten bis Zefau es korrigiert hat.

**Reply by kilian-f-za** [reply]
> Nein, man kann da nichts anders umsetzen. Man muss warten bis Zefau es korrigiert hat.

OK alles klar. Also werde ich erstmal alles so belassen
